documents/knowledge/processer.py

```diff
@@ -0,0 +1,43 @@
+import yaml
+
+import os
+import sys
+
+ELEMENTS_YAML = "elements.yaml"
+TEST_ELEMENTS_PY = "../../tests/test_elements.py"
+
+with open(ELEMENTS_YAML, "r", encoding="utf-8") as elements:
+    try:
+        data = yaml.safe_load(elements)
+    except yaml.YAMLError as ex:
+        print(ex)
```

Comment:
I don't see the point of catching and printing the error if you're going to exit anyway. Why not just let it be raised?

Reply:
It's dead code from when I was experimenting with getting the YAML processer working